platform.txt

```diff
@@ -2,6 +2,7 @@ name=ESP32 Arduino
 version=3.0.0
 
 tools.esp32-arduino-libs.path={runtime.platform.path}/tools/esp32-arduino-libs
+tools.esp32-arduino-libs.path.windows={runtime.platform.path}\tools\esp32-arduino-libs
```

Comment:
with that line added, it needs to be removed on release, because the whole line is overwritten: https://github.com/espressif/arduino-esp32/blob/master/.github/scripts/on-release.sh#L224

Please add the necessary sed command to clear that line when released

Reply:
Sorry, but I do need that line properly removed to accept this change.